VM/src/lvmexecute.cpp

```diff
@@ -1859,8 +1859,8 @@ static void luau_execute(lua_State* L)
                 else if (ttisvector(rb))
                 {
                     const float* vb = rb->value.v;
```

Comment:
```suggestion
                    const float* vb = vvalue(rb);
```
This is being fixed for all old instances internally.